content/docs/start/model-management/model-registry.md

```diff
@@ -7,149 +7,48 @@ description:
-filters, but they are hidden by default). The rest of the parameters are
-descriptive and optional and will also show up in DVC Studio.
-
-<details id="push-click-to-see-how-artifacts-are-registered">
-
-#### 💡 Expand to see how adding models to the registry works under the hood
-
-When we call the `log_artifact()` method, DVC takes all the information we
-provide in the call and edits the `dvc.yaml` file which will now contain the
-following lines:
-
-```yaml
-artifacts:
-  pool-segmentation:
-    path: models/model.pkl
-    type: model
-    desc:
-      This is a Computer Vision (CV) model that's segmenting out swimming pools
-      from satellite images.
-    labels:
-      - cv
-      - segmentation
-      - satellite-images
-```
-
-When you push the resulting file to your Git remote it is parsed by DVC Studio
-and it then shows all model artifacts from your `dvc.yaml` files in the model
-registry.
-
-</details>
-
-Now we just need to run the Python script which includes this code to cache and
-register the model. If you are
-[following](/doc/start/model-management/model-registry#follow-along-instructions)
-our example repository then this has already been done and we can continue to
-the next section.
-
-If you are building your own repository, you will need to run the script and
-push the result to your Git remote (e.g., GitHub) yourself.
-
-<details id="push-click-to-see-other-ways-to-add-models">
-
-#### 💡 Expand to see other ways to add models
-
-The other two options are to use the DVC Studio's graphical user interface to
-add models interactively or to manually edit `dvc.yaml` files to add information
-about model artifacts. To get more details on the ways to add models have a look
-at the
-[Model registry documentation](/doc/studio/user-guide/model-registry/add-a-model).
-
-</details>
+Using DVC and [DVC Studio](/doc/studio) we will set up a model registry where we
+can discover, share, deploy and audit all our models and which will serve as the
+single source of truth for our model management.
 
 ## DVC Model registry overview
 
 In this guide, we will be using [DVC Studio](https://studio.iterative.ai) to
-manage our model registry. DVC Studio enables you to see models across all
-projects, manage their lifecycle, and download them with only a token. You can
-find out more about it [here](/doc/studio).
-
-From the Models tab in DVC Studio we will have an overview of all models, latest
-model versions as well stages each of the model versions is assigned to. We can
-get more details for each model by clicking on the model name.
-
-You can check out our
+manage our model registry. You can check out our
 [example model](https://studio.iterative.ai/team/Iterative/models/b3P4bcYIrGYdzyjqzsf9Xw==/pool-segmentation/v1.0.0)
 in DVC Studio to see what it will look like once we finish all the steps in this
 guide.
 
-Now that we have added a model, you should see something like the following
```

Comment:
The rest of the changes on this page are just moving existing content around to fit the new structure